### ISSUE TITLE
Minor change to vi-github-issues.md

### ISSUE THREAD

**Issue by hiroTochigi** [open-learning-exchange/open-learning-exchange.github.io#2379]
### Problem
A link which explains the way to preview changes is a little out of date.
You cannot look at OLE Support page via Chrome without HTTP Server anymore (URL file://).
Also, the link indicates that the entry point of the OLE Support page is mdwiki.html, but open-learning-exchange.github.io does not have mdwiki.html but index.html  

### Screenshots

This is the link
![outdated-link](https://user-images.githubusercontent.com/44413241/59152246-dab0cc00-8a05-11e9-864f-274063db21dd.jpg)

Above link connect to this page
![outddated-page2](https://user-images.githubusercontent.com/44413241/59152261-1ea3d100-8a06-11e9-8b76-60f92ed97a00.png)

This screenshot is my repository; There is no mdwiki.html
![no-mdwiki](https://user-images.githubusercontent.com/44413241/59152298-a8ec3500-8a06-11e9-95e5-3e6880437133.png)

Google Chrome does not allow MDwiki to be rendered locally.
Have to render via localhost
![cannot-render](https://user-images.githubusercontent.com/44413241/59152362-fe751180-8a07-11e9-86c7-7471865a3ce6.png)

### Proposed solution

Disconnect the above link and write the explanation in this tutorial.

**Comment by hiroTochigi** [comment]
P.S.
My environment is Ubuntu Linux 18.06.
I verified that the OLE Support page was rendered by Firefox without HTTP Server and with HTTP Server, and Chrome with HTTP Server.
If I am allowed to deal with the issue, I will write an explanation in Firefox and Chrome.
If someone adds Microsoft Edge, Safari, and IE, it is great.

**Comment by Liew211** [comment]
@lmmrssa Can you take a look?

**Comment by lmmrssa** [comment]
@hiroTochigi That is link some external link and added there as reference. Which does not mean that every project will have **mdwiki.html** Also there is already mentioned that chrome will not work unless you open chrome with additional command.
Also there is another alternative give regarding online environment. So don't think we need to work on this.

**Comment by hiroTochigi** [comment]
Ok, I close this issue.

**Comment by hiroTochigi** [comment]
I have closed this issue.
However, I would like to put some improvement on the issue.
I was struggling to open the corrected file locally.
Even though we can open the file and examine the general Markdown expressions in [dillinger.io](https://dillinger.io/), we cannot examine the MDWiki's expressions.
First, I used Chrome, then I used FireFox. 
However, I could not open it because I did not know that I had to open it from **index.html** of the top directory: hiroTochigi.github.io in my case.
If people already know how MDWiki render the entire website, they can distinguish where the entry point is.
However, I could not figure it out at that time, so I think that someone might be confused about it.
